src/widgets/Tabs/PatientSelect.js

```diff
@@ -49,89 +55,50 @@ import { generateUUID } from '../../database/index';
 const PatientSelectComponent = ({
   createPatient,
   formConfig,
-  isAscending,
   onCancelPrescription,
   onFilterData,
-  onSortData,
   patients,
   selectPatient,
-  sortKey,
 }) => {
   const columns = React.useMemo(() => getColumns(MODALS.PATIENT_LOOKUP), []);
   const { pageTopViewContainer } = globalStyles;
-
-  const renderRow = React.useCallback(
-    listItem => {
-      const { item, index } = listItem;
-      const rowKey = item.id;
-      return (
-        <DataTableRow
-          rowData={patients[index]}
-          rowKey={rowKey}
-          getCallback={() => selectPatient(item)}
-          columns={columns}
-          rowIndex={index}
-          onPress={() => selectPatient(item)}
-        />
-      );
-    },
-    [patients]
-  );
-
-  const renderHeader = React.useCallback(
-    () => (
-      <DataTableHeaderRow
-        columns={columns}
-        isAscending={isAscending}
-        sortKey={sortKey}
-        onPress={onSortData}
-      />
-    ),
-    [columns, sortKey, isAscending]
-  );
+  const keyboardIsOpen = useKeyboardIsOpen();
 
   return (
-    <FlexView style={pageTopViewContainer}>
-      <AfterInteractions placeholder={null}>
-        <FlexRow style={{ marginBottom: 7 }} justifyContent="flex-end">
-          <VaccinePrescriptionInfo />
-        </FlexRow>
-
-        <View style={localStyles.container}>
-          <View style={localStyles.formContainer}>
-            <FormControl
-              inputConfig={formConfig}
-              onSave={onFilterData}
-              showCancelButton={false}
-              saveButtonText={generalStrings.search}
-            />
-          </View>
-          <View style={localStyles.verticalSeparator} />
-          <View style={localStyles.listContainer}>
-            <DataTable
-              data={patients}
-              columns={columns}
-              renderHeader={renderHeader}
-              renderRow={renderRow}
-              keyExtractor={item => item.id}
-              getItemLayout={getItemLayout}
-            />
+    <FlexView style={{ ...pageTopViewContainer, paddingVertical: 0 }}>
+      <Paper
+        style={{ flex: 6 }}
+        contentContainerStyle={{ flex: 1 }}
+        headerText={vaccineStrings.vaccine_dispense_step_one_title}
+      >
+        <AfterInteractions placeholder={null}>
+          <View style={localStyles.container}>
+            <View style={localStyles.formContainer}>
+              <FormControl
+                inputConfig={formConfig}
+                onSave={onFilterData}
+                showCancelButton={false}
+                saveButtonText={generalStrings.search}
+                showSaveButton={false}
+              />
+            </View>
+
+            <View style={localStyles.listContainer}>
+              <SimpleTable selectRow={selectPatient} data={patients} columns={columns} />
```

Comment:
controversial?

It's a fair amount simpler code but no sorting.. I mainly did it to get headers easily (they weren't showing up originally) and also because the data table has so much junky in it now it's  a little slower

Reply:
with no sorting for the user, can we sort the patient results first?
Current order is pretty odd - would look better if the results were in order.

With the search form I think dropping sort is ok